test/test-pages.js

```diff
@@ -0,0 +1,97 @@
+var fhir = require("../index.js");
+var expect  = require('chai').expect;
+
+describe('#fhirconvert()', function() {
+    context('Exponent', function() {
+      it('2 vars', function() {
+        expect(fhir.fhirconvert("a^b")).to.equal("(%a.power(%b))");
+      })
+      it('1 var v1', function() {
+        expect(fhir.fhirconvert("a^3")).to.equal("(%a.power(3))");
+      })
+      it('1 var v2', function() {
+        expect(fhir.fhirconvert("3^a")).to.equal("(3.power(%a))");
+      })
+      it('2 nums', function() {
+        expect(fhir.fhirconvert("3^3")).to.equal("(3.power(3))");
+      })
+      it('Decimals', function() {
+        expect(fhir.fhirconvert("3.3^4.2")).to.equal("(3.3.power(4.2))");
+      })
+      it('Parenthesis', function() {
+        expect(fhir.fhirconvert("(a+b)^(3+4)")).to.equal("((%a+%b).power((3+4)))");
+      })
+      it('Negative', function() {
+        expect(fhir.fhirconvert("-a^-3.3")).to.equal("(-%a.power(-3.3))");
+      })
+    })
+
+    context('Functions', function() {
+        it('CEILING', function() {
```

Comment:
I'm not sure if it's only my machine causing issues, but I'm getting 10 failing tests.

Reply:
I believe I just hashed out some of the issues that caused this.